### PR TITLE
Docs: Add caching strategies and strict equality comparison

### PR DESCRIPTION
**Pull request by fregante**
- Cross-repo PR: https://github.com/sindresorhus/mem/pull/83

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -33,6 +35,10 @@ const memGot = pMemoize(got, {maxAge: 1000});
 })();
 ```
 
+### Caching strategy
+
+See the [Caching strategy for `mem`](https://github.com/sindresorhus/mem#options).
```
Comment (fregante) [inline readme.md:40]: I added this instead of direct link so that the whole paragraph above can be copied whole.

**Comment by fregante** [comment]
Eeekkkk I was about to use a URL

**Comment by fregante** [comment]
Done directly on `main`

**Comment by sindresorhus** [comment]
I aint no mind reader

**Comment by fregante** [comment]
You should have known that my mouse was on the "Save changes" button. Isn't GitHub part of your being yet?

**Pull request merged**